src/base_obs.c

```diff
@@ -417,6 +418,13 @@ void base_obs_setup()
     &deprecated_callback,
     &deprecated_node
   );
+  
+  static sbp_msg_callbacks_node_t deprecated_node_2;
+  sbp_register_cbk(
+    SBP_MSG_OBS_DEP_B,
```

Comment:
After a deprecation period, we should be comfortable removing warnings. We did for the old unpacked obs messages. If it's not clear, I'm saying warn on `DEP_B`, not also `DEP_A` (note this is also generally a poor way to do "version" checking between devices).